docs/advanced-scenarios.md

```diff
@@ -0,0 +1,150 @@
+
+```javascript
+    {
+      pathSegment: 'Orders',
+      label: 'orders',
+      viewUrl: 'https://orders.microfrontend/',
+      navigationContext: 'orders',
+      virtualTree: true
+    }
+```
+
+2. Use an Angular Router for navigation.
+
+Angular provides [Router events](https://angular.io/guide/router#router-events). We are reacting on `NavigationEnd` to update the URL after a successful route change.
+
+We assume that the whole Angular app is one micro frontend and has its routes declared on the root level:
+
+```javascript
+  { path: 'preload', component: PreloadComponent },
+  { path: '', component: OrderListComponent },
+  { path: ':id', component: OrderComponent },
+  { path: ':id/details', component: OrderDetailsComponent },
+```
+
+Use this code to implement `luigi-auto-navigation.service.ts`, which is globally imported in our `app.module.ts`:
+
+```javascript
+    import { Router, NavigationEnd } from '@angular/router';
+    import { Injectable, OnDestroy } from '@angular/core';
+    import { Subscription } from 'rxjs';
+    import { filter } from 'rxjs/operators';
+    import { linkManager } from '@kyma-project/luigi-client';
+
+    @Injectable()
+    export class LuigiAutoNavigationService implements OnDestroy {
+      private subscriptions: Subscription = new Subscription();
+      constructor(private router: Router) {}
+      public init(): void {
+        this.subscriptions.add(
+          this.router.events
+            .pipe(filter(ev => ev instanceof NavigationEnd))
+            .subscribe((ev: NavigationEnd) => {
+              if (ev instanceof NavigationEnd) {
+                linkManager()
+                  .fromClosestContext()
+                  .withoutSync()
+                  .navigate(ev.url);
+              }
+            })
+        );
+      }
+
+      ngOnDestroy(): void {
+        this.subscriptions.unsubscribe();
+      }
+    }
+```
+
+#### Result
+
+Other than the added service, which could also be implemented as a `RouteGuard` or similar, the micro frontend is unchanged and uses `[routerLink='']` or other functionalities to navigate.
+
+### Authenticate Luigi with Google Cloud Identity
+
+#### Overview
+
+This example shows you how to use Luigi with a Google account.
+
+#### Steps
+
+1. Register a project and generate an OAuth2 Web Client based on [Google Developers Identity - OAuth2UserAgent](https://developers.google.com/identity/protocols/OAuth2UserAgent).
+2. To get your app running locally, set the Authorized JavaScript Origins URIs to `http://localhost:4200` and Authorized redirect URIs to `http://localhost:4200/luigi-core/auth/oauth2/callback.html?storageType=localStorage`.
+3. Copy the Client ID on the right side, ending with `apps.googleusercontent.com`
```

Comment:
```suggestion
3. Copy the Client ID on the right side, ending with `apps.googleusercontent.com`.
```
Is it the ClientID that already ends with `apps.googleusercontent.com` or does the user have to add it?  I assume it;'s the first one, judging from the code, but I think we might want to make it more precise.